README.md

```diff
@@ -40,6 +40,28 @@ Instead, it will copy all the configuration files and the transitive dependencie
 
 You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
+## Client-side CDC CDSi validator app
+
+Limited scope, CQL engine based client-side validation app that executes the CDSi supporting data guidelines.
+
+Currently only validates each non-Risk type series based on dose absMinAge, allowable vaccine codes (CVX only), and allowableInterval absMinInt fromPrevious.
+
+Does not support forecasting or any other validation, including conditional doses, recurring or seasonal doses, immunity or contradictions.
+
+### Update CQL library
+
+Script to convert CQL to ELM and write results to src/output-elm.
+
+`yarn translate`
```

Comment:
Since this requires the CQL translation service to be running on localhost it would be nice to have some mention of that here and instructions so that others can successfully run it.

Reply:
Added Docker pre-req. The translate script from https://github.com/mcode/cql-testing-harness does the work of downloading and spinning up the translator image if it is not running, so in theory no other setup required.

Reply:
oh cool I didn't realize that. I tried running locally, but must not have had Docker running.

Reply:
I think I was wrong about that actually - it looks like the harness only spins up the container if you run the full tests, not just translate to elm. I'll edit the README with the docker cmds.